### PR TITLE
Updated portuguese.json

### PR DESCRIPTION
**Pull request by DebeckGZN**
Added some translations

### CODE REVIEW

File: i18n/portuguese.json [hunk 1]
```diff
@@ -179,14 +179,14 @@
     "cmd-radio-play-help" : "Adds a radio to the queue to be played.",
     "cmd-radio-list-help" : "Displays the list of my available radio stations.",
     "cmd-search-help" : "Searches multiple videos at once with the selected parameters.\nYou can add one of them to the queue. If no service are specified, YouTube is selected by default.\n\nList of the services (platform where the video is searched) :\n- YouTube (yt) (by default)\n- SoundCloud (sc)",
-    "cmd-moveto-help" : "Moves the song to a specific moment.",
-    "cmd-download-help" : "Allows to create a download link for a YouTube video.\nIf don't specify an URL, for the current song.",
```
Comment (Zenrac) [inline i18n/portuguese.json]: Can you translate the end of the line ?

File: i18n/portuguese.json [hunk 2]
```diff
@@ -179,14 +179,14 @@
...
+    "cmd-moveto-help" : "Move a música para um momento específico.",
+    "cmd-download-help" : "Permite criar um link de download de um vídeo do Youtube.\nIf don't specify an URL, for the current song.",
```
Comment (Zenrac) [inline i18n/portuguese.json]: Can you translate the end of the line ?

Comment (DebeckGZN) [reply i18n/portuguese.json]: sure!

**Comment by DebeckGZN** [comment]
I have finished the line and added some new ones too!

**Pull request merged**